icechunk/src/cli/default.rs

```diff
@@ -0,0 +1,58 @@
+use crate::{
+    cli::config::{RepoLocation, RepositoryDefinition, S3Credentials},
+    config::S3Options,
+    RepositoryConfig,
+};
+
+pub fn create_s3_repo() -> RepositoryDefinition {
+    RepositoryDefinition::S3 {
+        location: RepoLocation {
+            bucket: "testbucket".to_string(),
+            prefix: "testfolder".to_string(),
+        },
+        object_store_config: S3Options {
+            region: Some("eu-west-1".to_string()),
+            endpoint_url: None,
+            anonymous: false,
+            allow_http: false,
+        },
+        credentials: S3Credentials::FromEnv,
+        config: RepositoryConfig::default(),
+    }
+}
+
+pub fn create_local_repo() -> RepositoryDefinition {
+    RepositoryDefinition::LocalFileSystem {
+        path: std::path::PathBuf::from("testfolder"),
+        config: RepositoryConfig::default(),
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use std::{collections::HashMap, fs::File};
+
+    use serde_yaml_ng::{from_reader, to_writer};
+
+    use crate::cli::config::{Repositories, RepositoryAlias};
+
+    use super::*;
+
+    #[test]
+    fn test_generate_repos() {
+        let s3_repo = create_s3_repo();
+        let local_repo = create_local_repo();
+
+        let mut repos = Repositories { repos: HashMap::new() };
+
+        repos.repos.insert(RepositoryAlias("s3repo".to_string()), s3_repo);
+        repos.repos.insert(RepositoryAlias("localrepo".to_string()), local_repo);
+
+        let path = "default.yaml";
```

Comment:
I don't know ... this test is writing code to the repos itself. Maybe move it to a temp file?

Reply:
Absolutely, was just a quick way of generating something.

I have added a rudimentary `config init` in https://github.com/earth-mover/icechunk/pull/716/commits/4d7afdaa2115772accc78ba12ad9f8cef65c4ad3